hack/record-gcp

```diff
@@ -1,44 +1,17 @@
 #!/bin/bash
 
-set -e
-set -x
+set -o errexit
+set -o nounset
+set -o pipefail
 
-export KUBEBUILDER_ASSETS=$(go run sigs.k8s.io/controller-runtime/tools/setup-envtest@latest use -p path)
+REPO_ROOT="$(git rev-parse --show-toplevel)"
+cd ${REPO_ROOT}
 
-rm -rf $(pwd)/artifactz/realgcp
-
-RUN_TESTS=TestAllInSeries/$1
-
-if [[ -z "${GCP_PROJECT_ID:-}" ]]; then
-  GCP_PROJECT_ID=$(gcloud config get-value project)
-fi
-echo "Using project: ${GCP_PROJECT_ID}"
-
-TEST_FOLDER_ID=$(gcloud projects describe ${GCP_PROJECT_ID} --format='value(parent.id)')
-export TEST_FOLDER_ID
+# Run the tests specified on the command line
+export RUN_TESTS=TestAllInSeries/$1
 
-TEST_ORG_ID=$(gcloud projects get-ancestors ${GCP_PROJECT_ID} | grep organization | cut -f1 -d ' ')
-export TEST_ORG_ID
-
-if [[ -z "${TEST_BILLING_ACCOUNT_ID:-}" ]]; then
-  TEST_BILLING_ACCOUNT_ID=$(gcloud billing projects describe ${GCP_PROJECT_ID} --format='value(billingAccountName)' | cut -f2 -d/)
-fi
-echo "TEST_BILLING_ACCOUNT_ID=${TEST_BILLING_ACCOUNT_ID}"
-export TEST_BILLING_ACCOUNT_ID
-
-if [[ -z "${KCC_USE_DIRECT_RECONCILERS:-}" ]]; then
-  KCC_USE_DIRECT_RECONCILERS="ComputeForwardingRule"
-fi
-export KCC_USE_DIRECT_RECONCILERS
-echo "KCC_USE_DIRECT_RECONCILERS=$KCC_USE_DIRECT_RECONCILERS"
+# Clear existing artifacts
```

Comment:
I think we are losing the `KCC_USE_DIRECT_RECONCILERS` here

Reply:
run-e2e sets this variable (line 27), and has more items in that list (so I think the two got out of sync, which is probably not right).